docker-compose.yml

```diff
@@ -11,21 +11,21 @@ services:
 #    container_name: frontier
 #    command: rocksdb.path=/crawldir/rocksdb
 #    ports:
-#      - 7071:7071
+#      - "127.0.0.1:7071:7071"
```

Comment:
for my own education - what is the benefit if specifying 127.0.0.1?

Reply:
If you are not using localhost, docker will interfer with any active firewall and expose the port "7071" on any available network interface (regardless if you have configured a firewall to block all ports externally).